README.md

```diff
@@ -1,6 +1,34 @@
 # hmpps-template-typescript
 Template github repo used for new Typescript based projects.
 
+# Instructions
+
+If this is a Digital Prison Services project then the project will be created as part of bootstrapping - 
+see https://github.com/ministryofjustice/dps-project-bootstrap.
```

Comment:
isn't this project now used by lots of probation people too?

Reply:
updated.